service/src/test/java/tds/assessment/repositories/impl/AssessmentCommandRepositoryImplIntegrationTests.java

```diff
@@ -0,0 +1,74 @@
+/***************************************************************************************************
+ * Copyright 2017 Regents of the University of California. Licensed under the Educational
+ * Community License, Version 2.0 (the “license”); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the license at
+ *
+ * https://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required under applicable law or agreed to in writing, software distributed under the
+ * License is distributed in an “AS IS” BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for specific language governing permissions
+ * and limitations under the license.
+ **************************************************************************************************/
+
+package tds.assessment.repositories.impl;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.io.Resource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import tds.assessment.Assessment;
+import tds.assessment.repositories.AssessmentCommandRepository;
+import tds.assessment.repositories.AssessmentQueryRepository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest
+@Transactional
+public class AssessmentCommandRepositoryImplIntegrationTests {
+    private static final String TEST_CLIENT_NAME = "SBAC_PT";
+    private static final String TEST_ASSESSMENT_KEY = "(SBAC_PT)SBAC-IRP-CAT-ELA-3-Summer-2015-2016";
+
+    @Autowired
+    private AssessmentCommandRepository commandRepository;
+
+    @Autowired
+    private AssessmentQueryRepository queryRepository;
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @Value("classpath:(SBAC_PT)SBAC-IRP-CAT-ELA-3-Summer-2015-2016.xml")
+    private Resource res;
+
+    @Before
+    public void setup() throws IOException {
+        // Loads the test package
+        jdbcTemplate.execute(String.format("CALL itembank.loader_main('%s')", Files.toString(res.getFile(), Charsets.UTF_8)));
```

Comment:
If this test fails for any reason is this cleaned up?

Reply:
Actually, how does this test run correctly?  For example, flyway doesn't create these stored procedures.  So how will they get on the dbs?  Or are we saying that everyone should make sure to run the db insert scripts on local?

If so, make sure to run that on the TeamCity database before merging this PR

Reply:
If the test fails, things are cleaned up by Spring automatically because of the class-level `@Transactional` tag. But you do bring up a good point in your second comment. The `loader_main()` and other loader stored procedures may need to be added. This worked for me locally because I have all the stored procedures/tables/schemas created in my local mysql db.

Reply:
Ah, I missed the class level tag.  To the second point, what if we added the SQL dump to the flyway migration files?  This would include the table structure, stored procedures, and functions.  We could do that in every project if we wanted.  

That would give us two things:
1. It wouldn't force someone to manually run the schema themselves locally and could just use flyway
2. It would set this project up for the import work as whoever does that will need to do to the work of adding tests for import which can't use the stored procedure.

Whenever the create new procedure work is done I would expect this test to be modified to not use the loader_main procedures.  I think those procedures should be removed once this amendment is complete.